cli/command/stack/deploy_composefile_test.go

```diff
@@ -25,13 +26,29 @@ services:
 	file := tempfile.NewTempFile(t, "test-get-config-details", content)
 	defer file.Remove()
 
-	details, err := getConfigDetails(file.Name())
+	details, err := getConfigDetails(file.Name(), nil)
 	require.NoError(t, err)
 	assert.Equal(t, filepath.Dir(file.Name()), details.WorkingDir)
 	assert.Len(t, details.ConfigFiles, 1)
 	assert.Len(t, details.Environment, len(os.Environ()))
 }
 
+func TestGetConfigDetailsStdin(t *testing.T) {
+	content := `
+version: "3.0"
+services:
+foo:
+image: alpine:3.5
```

Comment:
Looks like the indentation got lost here. We're not testing if the content is actually parsed, so won't make a difference (currently); I see the other test also doesn't do this, but wondering if we should, e.g.;

```patch
diff --git a/cli/command/stack/deploy_composefile_test.go b/cli/command/stack/deploy_composefile_test.go
index 90d866a2..6e2a1f2c 100644
--- a/cli/command/stack/deploy_composefile_test.go
+++ b/cli/command/stack/deploy_composefile_test.go
@@ -37,15 +37,16 @@ func TestGetConfigDetailsStdin(t *testing.T) {
        content := `
 version: "3.0"
 services:
-foo:
-image: alpine:3.5
+  foo:
+    image: alpine:3.5
 `
        details, err := getConfigDetails("-", strings.NewReader(content))
        require.NoError(t, err)
        cwd, err := os.Getwd()
        require.NoError(t, err)
        assert.Equal(t, cwd, details.WorkingDir)
-       assert.Len(t, details.ConfigFiles, 1)
+       require.Len(t, details.ConfigFiles, 1)
+       assert.Equal(t, "3.0", details.ConfigFiles[0].Config["version"])
        assert.Len(t, details.Environment, len(os.Environ()))
 }
```